.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: xenial
```

Comment:
This seems to be completely out of place, and not in line with PM and NicoSetton's comment on Slack. Why not let GPS's CI take care of this ?

Reply:
After discussion w. Anthony live: You guys discussed that this was a good compromise. Carry on :)